### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by UmeshKonduru**
Added more clarity for beginners

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,11 @@ More details at [kilvish.in](https://kilvish.in)
...
    - [Download flutter](https://docs.flutter.dev/get-started/install)
    - [Setup Editor](https://docs.flutter.dev/get-started/editor?tab=vscode)
-   - Run `flutter doctor` in your PC terminal to figure out anything that flutters need that is not installed on your PC
+   - Run `flutter doctor` in your PC terminal to figure out anything that flutter needs that is not installed on your PC.
 2. Run code
-   - `flutter pub get` to install dependencies. Your editor will do it by default once you have cloned the repository & opened it in the editor
+   - Run `git clone https://github.com/tamraj-kilvish/kilvish.git` in your PC terminal to clone the repository.
+   - Navigate to the directory of the cloned repository in your PC terminal.
+   - `flutter pub get` to install dependencies. Your editor will do it by default once you have cloned the repository & opened it in the editor.
    - `flutter run` will launch the app in debug mode. You can also launch it from Run command of your editor.
```
Comment (pocha) [inline README.md:22]: Maybe we remove the parts like 'Your editor will do ... ' & 'You can also launch it ...' & just add one more line at the end which says - if you using VisualStudio or some other editor, they might have inbuilt capacity to do 'flutter pub get' & 'flutter run'. Eg. VisualStudio runs the app with Run command if you have the flutter extension installed.

Comment (UmeshKonduru) [reply README.md:22]: Sure we can do that

File: README.md [hunk 2]
```diff
@@ -14,10 +14,13 @@ More details at [kilvish.in](https://kilvish.in)
 1. Setup Flutter on your PC
    - [Download flutter](https://docs.flutter.dev/get-started/install)
    - [Setup Editor](https://docs.flutter.dev/get-started/editor?tab=vscode)
-   - Run `flutter doctor` in your PC terminal to figure out anything that flutters need that is not installed on your PC
+   - Run `flutter doctor` in your PC terminal to figure out anything that flutter needs that is not installed on your PC.
 2. Run code
-   - `flutter pub get` to install dependencies. Your editor will do it by default once you have cloned the repository & opened it in the editor
-   - `flutter run` will launch the app in debug mode. You can also launch it from Run command of your editor.
+   - Run `git clone https://github.com/tamraj-kilvish/kilvish.git` in your PC terminal to clone the repository.
+   - Navigate to the directory of the cloned repository in your PC terminal.
```
Comment (Saga-sanga) [inline README.md:20]: Run 'cd <directory_name>' to navigate to the cloned directory in your PC.

**Review by Saga-sanga — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (UmeshKonduru) [reply README.md:20]: Okay

File: README.md [hunk 3]
```diff
@@ -14,10 +14,13 @@ More details at [kilvish.in](https://kilvish.in)
...
+   - `flutter pub get` to install dependencies.
```
Comment (pratik9333) [inline README.md:21]: I would recommend to remove flutter pub get line. Reason is it is doing by default so no need to be mention in detail.

Comment (pratik9333) [reply README.md:21]: Keep flutter run simple.

**Review by pratik9333 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (UmeshKonduru) [reply README.md:21]: Not everyone uses Visual Studio. I guess it doesn't hurt to keep it.

Comment (pratik9333) [reply README.md:21]: Every ide does it by default. Dart has some extentions that is installed by default that runs flutter pub command.

Comment (UmeshKonduru) [reply README.md:21]: Everyone doesn't use an IDE. For instance, I'm using a lightweight editor right now. Again, I don't see any problem with keeping the line - it doesn't take much effort to run the command.

**Review by pratik9333 — APPROVED** [review]

**Comment by pratik9333** [comment]
@UmeshKonduru You can merge now

**Pull request merged**